Jenkinsfile

```diff
@@ -174,12 +174,8 @@ pipeline {
 
     // trigger the other job to update the upstream reference
     stage ('Trigger Enterprise Update') {
-      when {
-        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
-      }
-
       steps {
-        build job: "frontend/dcos-ui-ee-release/${env.BRANCH_NAME}", parameters: [[$class: 'StringParameterValue', name: 'BRANCH_NAME', value: env.BRANCH_NAME]]
+        build job: "frontend/dcos-ui-ee-pipeline/${env.BRANCH_NAME}", parameters: [[$class: 'StringParameterValue', name: 'BRANCH_NAME', value: env.BRANCH_NAME]]
```

Comment:
I think we need to run it all the time because we don't wanna break plugins unintentionally.